doc/classes/TubeTrailMesh.xml

```diff
@@ -1,8 +1,10 @@
 <?xml version="1.0" encoding="UTF-8" ?>
 <class name="TubeTrailMesh" inherits="PrimitiveMesh" version="4.1" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:noNamespaceSchemaLocation="../class.xsd">
 	<brief_description>
+		Represents a straight tube-shaped [PrimitiveMesh] with variable width.
 	</brief_description>
 	<description>
+		[TubeTrailMesh] represents a straight tube-shaped mesh with variable width. The tube is composed of a number of cylindrical sections, each with the same [member section_length] and number of [member section_rings]. A [member curve] is sampled along the total length of the tube, meaning that the curve determines the radius of the tube along its length.
 	</description>
 	<tutorials>
```

Comment:
Should link to https://docs.godotengine.org/en/latest/tutorials/3d/particles/trails.html and https://docs.godotengine.org/en/latest/tutorials/3d/particles/index.html